orchestrator/ethereum_gravity/src/utils.rs

```diff
@@ -123,6 +124,26 @@ pub async fn get_gravity_id(
     }
 }
 
+pub async fn get_gravity_sol_address(
```

Comment:
this should have a doc comment describing what it does. It's for getting the gravity.sol address from gravity_erc721, this address is the only address allowed to call the withdraw function

Reply:
Comment has been added